crates/node/src/networking/p2p.rs

```diff
@@ -175,9 +313,141 @@ mod tests {
         }
     }
 
+    #[tokio::test]
+    async fn test_peer_list_inter_connection() {
+        //start_logger(LevelFilter::ERROR);
```

Comment:
```suggestion
        start_logger(LevelFilter::ERROR);
```
In the merged version, I'd prefer debug logs silenced, even if the normal `cargo test` hides the output. WDYT?

Reply:
I remove it because in the CI you don't need it and as all test load it if you start all the test in //  it breaks because it's already loaded by another test.
You run the test one after the other in the CI ?

Reply:
> I remove it because in the CI you don't need it and as all test load it if you start all the test in // it breaks because it's already loaded by another test.

Ah! I wasn't aware of this. Zap it away then :+1:

> You run the test one after the other in the CI ?

The CI runs the tests with the default settings of `cargo test`.

Reply:
Strange if I run cargo test locally I have this error: 
```
Unable to install global subscriber: SetGlobalDefaultError("a global default trace dispatcher has already been set")
```
I've to remove 2 from 3 to pass the test. 
tracing_subscriber::init() must be call only one time per execution. Perhaps the CI start an exe for each test.
But local execution break with the logger inited for each test.
I'm not aware of the last dev, but I think that the default Rust unit test executor don't have per or post execution for all test.
You really need login for the test?

Reply:
> You really need login for the test?

No, not until debugging the test runs :slightly_smiling_face: 

> Zap it away then
By this I mean: Go ahead and delete the logging setup calls. Shouldn't that fix it?

Reply:
Usually I left the comment so that when you need to debug it's easier to activate.